blocks/editable/index.js

```diff
@@ -119,7 +119,7 @@ export default class Editable extends Component {
 		editor.on( 'keyup', this.onKeyUp );
 		editor.on( 'selectionChange', this.onSelectionChange );
 		editor.on( 'BeforeExecCommand', this.maybePropagateUndo );
-		editor.on( 'BeforePastePreProcess', this.onBeforePastePreProcess );
+		editor.on( 'PastePreProcess', this.onPastePreProcess, true /* Add before core handlers */ );
```

Comment:
Why was this change necessary?

Reply:
The `BeforePastePreProcess` event has been removed.